### ISSUE TITLE
Explore Semantic Scholar API for Subject Area / Topic Discovery

### ISSUE THREAD

**Issue by de-code** [sciety/sciety#1265]
This issue is to collect some information relating to the Semantic Scholar API that may be useful for subject area / topic discovery.

**Comment by de-code** [comment]
The API is fairly well documented at https://www.semanticscholar.org/product/api and https://api.semanticscholar.org/graph/v1

There is the "old" paper API, e.g. https://api.semanticscholar.org/v1/paper/10.1038/nrn3241

<details>
<summary>full response (with only first citation and first reference)</summary>

```json
{
  "abstract": "Neuronal activity in the brain gives rise to transmembrane currents that can be measured in the extracellular medium. Although the major contributor of the extracellular signal is the synaptic transmembrane current, other sources \\u2014 including Na+ and Ca2+ spikes, ionic fluxes through voltage- and ligand-gated channels, and intrinsic membrane oscillations \\u2014 can substantially shape the extracellular field. High-density recordings of field activity in animals and subdural grid recordings in humans, combined with recently developed data processing tools and computational modelling, can provide insight into the cooperative behaviour of neurons, their average synaptic input and their spiking output, and can increase our understanding of how these processes contribute to the extracellular signal.",
  "arxivId": null,
  "authors": [
    {
      "authorId": "144128278",
      "name": "G. Buzs\\u00e1ki",
      "url": "https://www.semanticscholar.org/author/144128278"
    },
    {
      "authorId": "2277912",
      "name": "C. Anastassiou",
      "url": "https://www.semanticscholar.org/author/2277912"
    },
    {
      "authorId": "145624227",
      "name": "C. Koch",
      "url": "https://www.semanticscholar.org/author/145624227"
    }
  ],
  "citationVelocity": 352,
  "citations": [
    {
      "arxivId": null,
      "authors": [
        {
          "authorId": null,
          "name": "Roxanne Lofredi"
        },
        {
          "authorId": null,
          "name": "Andrea A. K\\u00fchn"
        }
      ],
      "doi": "10.1016/b978-0-12-821861-7.00007-5",
      "intent": [],
      "isInfluential": false,
      "paperId": "784a3b560ba87a816a54567677839b7962003163",
      "title": "Electrophysiological connectivity measures from deep brain stimulation (DBS)-targets in Parkinson\\u2019s disease and dystonia",
      "url": "https://www.semanticscholar.org/paper/784a3b560ba87a816a54567677839b7962003163",
      "venue": "Connectomic Deep Brain Stimulation",
      "year": 2022
    },
    ...
  ],
  "corpusId": 205507595,
  "doi": "10.1038/nrn3241",
  "fieldsOfStudy": [
    "Biology",
    "Medicine"
  ],
  "influentialCitationCount": 147,
  "isOpenAccess": true,
  "isPublisherLicensed": true,
  "is_open_access": true,
  "is_publisher_licensed": true,
  "numCitedBy": 2655,
  "numCiting": 216,
  "paperId": "da82f8e6ff009432896730061247fa6653bed1f0",
  "references": [
    {
      "arxivId": null,
      "authors": [
        {
          "authorId": "52254908",
          "name": "Mmh"
        }
      ],
      "doi": "10.1097/00005053-194508000-00033",
      "intent": [],
      "isInfluential": false,
      "paperId": "20b28fd3150c9075f8de2c8c110d2ec069bd6e3b",
      "title": "Studies from the Rockefeller Institute for Medical Research",
      "url": "https://www.semanticscholar.org/paper/20b28fd3150c9075f8de2c8c110d2ec069bd6e3b",
      "venue": "",
      "year": 1945
    },
    ...
  ],
  "title": "The origin of extracellular fields and currents \\u2014 EEG, ECoG, LFP and spikes",
  "topics": [
    {
      "topic": "Electrocorticogram",
      "topicId": "43814",
      "url": "https://www.semanticscholar.org/topic/43814"
    },
    {
      "topic": "Sodium",
      "topicId": "1187",
      "url": "https://www.semanticscholar.org/topic/1187"
    },
    {
      "topic": "Electroencephalography",
      "topicId": "239",
      "url": "https://www.semanticscholar.org/topic/239"
    },
    {
      "topic": "Subdural space",
      "topicId": "2371",
      "url": "https://www.semanticscholar.org/topic/2371"
    },
    {
      "topic": "Ligand-Gated Ion Channels",
      "topicId": "862352",
      "url": "https://www.semanticscholar.org/topic/862352"
    },
    {
      "topic": "Ligands",
      "topicId": "158939",
      "url": "https://www.semanticscholar.org/topic/158939"
    },
    {
      "topic": "Tissue membrane",
      "topicId": "1269",
      "url": "https://www.semanticscholar.org/topic/1269"
    }
  ],
  "url": "https://www.semanticscholar.org/paper/da82f8e6ff009432896730061247fa6653bed1f0",
  "venue": "Nature Reviews Neuroscience",
  "year": 2012
}
```
</details>

The response shows two relevant fields: `fieldsOfStudy` and `topics`

```json
{
  "fieldsOfStudy": [
    "Biology",
    "Medicine"
  ],
  "topics": [
    {
      "topic": "Electrocorticogram",
      "topicId": "43814",
      "url": "https://www.semanticscholar.org/topic/43814"
    },
    {
      "topic": "Sodium",
      "topicId": "1187",
      "url": "https://www.semanticscholar.org/topic/1187"
    },
    {
      "topic": "Electroencephalography",
      "topicId": "239",
      "url": "https://www.semanticscholar.org/topic/239"
    },
    {
      "topic": "Subdural space",
      "topicId": "2371",
      "url": "https://www.semanticscholar.org/topic/2371"
    },
    {
      "topic": "Ligand-Gated Ion Channels",
      "topicId": "862352",
      "url": "https://www.semanticscholar.org/topic/862352"
    },
    {
      "topic": "Ligands",
      "topicId": "158939",
      "url": "https://www.semanticscholar.org/topic/158939"
    },
    {
      "topic": "Tissue membrane",
      "topicId": "1269",
      "url": "https://www.semanticscholar.org/topic/1269"
    }
  ],
  ...
}
```

The topic url leads to a web page rather than an api.

The graph paper API only returns very few fields by default. Instead one needs to specify the fields to return. Currently the available fields do not seem to include the topics. It is currently:

- paperId - Always included
- externalIds
- url
- title - Included if no fields are specified
- abstract
- venue
- year
- referenceCount
- citationCount
- influentialCitationCount
- isOpenAccess
- fieldsOfStudy
- authors - ...
- citations - ...
- references - ...
- embedding
- tldr

Example URL: https://api.semanticscholar.org/graph/v1/paper/649def34f8be52c8b66281af98ae884c09aef38b?fields=title,embedding

**Comment by de-code** [comment]
Tested the availability of our top 10 papers in Sciety:

```text
10.1101/2021.07.05.451181
10.1101/2021.08.10.455713
10.1101/2021.05.28.446245
10.1101/2021.07.13.21260393
10.1101/2021.08.14.21262020
10.1101/2021.06.18.449063
10.1101/2021.03.21.436299
10.1101/2021.05.29.21258059
10.1101/2021.08.04.454987
10.1101/2021.09.09.459577
```

e.g.

```bash
curl https://api.semanticscholar.org/v1/paper/10.1101/2021.07.05.451181 | jq .topics
```

For all of them, `topics` is an empty array.

Also tested `10.7554/eLife.67490` to use a DOI with a different prefix with the same result.

This suggests that the topics are only available for a selected number of papers. Perhaps that is why they are not currently included in the graph API.

**Comment by de-code** [comment]
Regarding `fieldsOfStudy`:

```bash
curl https://api.semanticscholar.org/graph/v1/paper/10.1101/2021.07.05.451181?fields=fieldsOfStudy | jq .fieldsOfStudy
```

For those papers the `fieldsOfStudy` was either `["Biology"]` or `["Medicine"]`.
